src/routes/event-match.tsx

```diff
@@ -142,15 +189,33 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
       }
       class={clsx(
         matchStyle,
-        match && reports && loadedMatchStyle,
-        match?.videos && match.videos.length > 0 && matchWithVideoStyle,
+        match && offlineMatchStyle,
+        match && isOnline && loadedMatchStyle,
+        match?.videos &&
+          match.videos.length > 0 &&
+          isOnline &&
+          matchWithVideoStyle,
       )}
     >
-      {match && reports ? (
+      {match ? (
         <>
+          {!isOnline && (
+            <div class={offlineDisplayInfo}>
+              <Card
+                class={css`
+                  justify-self: center;
+                  padding: 1rem;
+                  text-align: center;
+                `}
+              >
```

Comment:
Would it work to combine the outer div and the card into just the card?

```js
<Card class={offlineDisplayInfo}>
```
And move the CSS properties from the inline block up to the offlineDisplayInfo style block?

Reply:
On screen sizes < 540px and > ~500px, the edges of the offline card don't line up with the edges of everything else. I think combining the outer div with the card may fix that, but in case it doesn't, can you fix that as well?